common/txmgr/types/chain.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import "fmt"
+
+// Sequence represents the base type, for any chain's sequence object.
+// It should be convertible to a string
+type Sequence fmt.Stringer
+
+// Id represents the base type, for any chain's Id.
+// It should be convertible to a string, that can uniquely identify this chain
+type Id fmt.Stringer
```

Comment:
ID should still be caps according to Go naming, since it's an acronym: 
```suggestion
// ID represents the base type, for any chain's ID.
// It should be convertible to a string, that can uniquely identify this chain
type ID fmt.Stringer
```

Reply:
done